lib/storage/data/external/GcpClient.js

```diff
@@ -298,6 +304,42 @@ class GcpClient extends AwsClient {
             return callback();
         });
     }
+
+    listObjects(params, callback) {
```

Comment:
why do we need these extra methods?
the required methods should be already implemented in `AwsClient`...